apiserver/stateauthenticator/delegator.go

```diff
@@ -35,11 +35,11 @@ func (p *PermissionDelegator) SubjectPermissions(
 		return permission.NoAccess, errors.Trace(err)
 	}
 
-	access, err := p.AccessService.ReadUserAccessForTarget(context.TODO(), userID, permissionID)
+	access, err := p.AccessService.ReadUserAccessLevelForTarget(context.TODO(), userID, permissionID)
```

Comment:
We need to come back around to this and thread context through from API calls.

Reply:
Got a task for that in my Jira